localisation/00_zuwc_united_localisation.csv

```diff
@@ -35,7 +35,6 @@ EVTDESC_WCFTW_101;As the great war between the Orcish Horde and the kingdom of S
 EVTOPTA_WCFTW_101;We're not going to send our troops to someone else's war.;;Wir werden unsere Truppen nicht in einen fremden Krieg schicken.;;;;;;;;;;;x
 EVTOPTB_WCFTW_101;This does not concern us.;�a ne nous concerne pas.;Das geht uns nichts an.;;;;;;;;;;;x
 EVTDESC_WCFTW_102;Today, the infantry divisions of the dwarves of Ironforge arrived. Perhaps not all is lost...;;Heute kamen in der Hauptstadt die Infanteriedivisionen der Zwerge von Ironforge an. Vielleicht ist doch noch nicht alles verloren...;;;;;;;;;;;x
-EVTOPTC_WCFTW_100;Excellent!;Excellent�!;Gro�artig!;;;;;;;;;;;x
```

Comment:
shouldn't this be wcftw_102?

Reply:
Fixed in 23b3a49d4f2dfd54f9aef6088c626e3c887efdc1.